### PR TITLE
Add PAWS to Tools Directory

### PR DESCRIPTION
**Pull request by niedzielski**
Complete step #1 of adding a new project to Tools Directory[0]. Step #2
- 3 will point to the GitHub repo once this is merged.

[0] https://tools.wmflabs.org/hay/directory/?search=paws#addtool

### CODE REVIEW

**Comment by niedzielski** [comment]
p/ Let me know if there's anything you want changed. For instance, I think this toolinfo.json could live in a subdir too.

File: toolinfo.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "name" : "paws",
+    "title" : "PAWS",
+    "description" : "Pywikibot: A Web Shell",
```
Comment (yuvipanda) [inline toolinfo.json]: It is now 'PAWS: A Web Shell' :D

**Comment by yuvipanda** [comment]
Thank you so much for the PR! I made one comment - if you fix that I'll merge!

**Comment by niedzielski** [comment]
:+1: Ok, I think I have it now. Side note: don't forget to update your Git one liner for the repo! :]

**Comment by yuvipanda** [comment]
\o/ done! Thanks for the patch

**Comment by niedzielski** [comment]
:+1: Ok, the remaining steps are done. PAWS is now discoverable via https://tools.wmflabs.org/hay/directory/?search=paws#/search/python :)

**Comment by yuvipanda** [comment]
\o/ Awesome! Thanks for taking this on!

On Sun, Jul 3, 2016 at 6:59 PM, Stephen Niedzielski <
notifications@github.com> wrote:

> 👍 Ok, the remaining steps are done. PAWS is now discoverable via
> https://tools.wmflabs.org/hay/directory/?search=paws#/search/python :)
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/yuvipanda/paws/pull/11#issuecomment-230163351>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AAB23hrqJxIz-F_lj0dy_Un3EdFNdZkwks5qR-p2gaJpZM4JCyxw>
> .
>



-- 
Yuvi Panda T
http://yuvi.in/blog

**Pull request merged**